src/Simulation/TargetDefinitions/Decompositions/RzzFromExp.qs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+    open Microsoft.Quantum.Targeting;
+
+    /// # Summary
+    /// Applies the two qubit Ising $ZZ$ rotation gate.
+    ///
+    /// # Description
+    /// \begin{align}
+    ///     R_zz(\theta) \mathrel{:=}
+    ///     \begin{bmatrix}
+    ///         e^{-i \theta / 2} & 0 & 0 & 0 \\\\
```

Comment:
(May be false alarm)
The matrix on [this page](https://www.numerade.com/ask/question/in-some-quantum-computing-platforms-the-easiest-two-qubit-gate-to-implement-is-something-called-the-ising-zz-coupling-gate-in-the-standard-basis-this-kind-of-gate-is-represented-by-matrix-of-33007/) makes me suspect that the following can be applicable 
```suggestion
    ///         e^{i \theta / 2} & 0 & 0 & 0 \\\\
```

Reply:
Applicable to more than one file.

Reply:
The matrices have been validated with the hardware teams of our partners, exactly because these kinds of angle conventions vary and can cause problems when different in the language vs the hardware. For this, we are following the conventions and matrices that do have the negative sign in the exponent on the first row, as shown here: https://en.wikipedia.org/wiki/List_of_quantum_logic_gates#Two_qubit_interaction_gates.

Reply:
(Just FYI)
I have an impression that the implementation is correct but the doc comment line that I refer to here is still not quite right.

When I look at the [link that I provided](https://www.numerade.com/ask/question/in-some-quantum-computing-platforms-the-easiest-two-qubit-gate-to-implement-is-something-called-the-ising-zz-coupling-gate-in-the-standard-basis-this-kind-of-gate-is-represented-by-matrix-of-33007/) and the [Wikipedia link that you provided](https://en.wikipedia.org/wiki/List_of_quantum_logic_gates#Two_qubit_interaction_gates) (I assume you mean the line containing "ZZ interaction") I see that in both cases the matrix consists of the main diagonal (all other elements are 0), and
* the first-most and the last-most element in the matrix have the same sign
* and the 2 middle elements have the opposite sign.

But in your doc-comment 3 elements of the matrix have one sign, and one element has opposite sign.

In other words, in both cases of the documentation 2 elements have one sign and 2 elements have the other sign.
But in your doc-comment 3 elements have one sign and 1 element has the other sign.

If your doc-comment is still correct, and the implementation corresponds to that doc-comment, then I suspect there was a bug when "The matrices have been validated with the hardware teams of our partners".

(this logic can still be a false alarm, just FYI)

Reply:
Ah, you're right! The upper left corner is correct, but the rest are mismatched for each of the copy pasted `Rzz` doc comments. I'll get that fixed!

Reply:
Just in case I'll remind: Applicable to more than one file.

Reply:
Oops! Looks like you have already taken care of multiple files.